doc/modules/gaussian_process.rst

```diff
@@ -480,8 +492,9 @@ shown in the following figure:
    :target: ../auto_examples/gaussian_process/plot_gpr_prior_posterior.html
    :align: center
 
-References
-----------
+|details-start|
+**References**
+|details-split|
```

Comment:
Unfortunately these references are called just above the Gaussian Process Kernel API section and they got broken by the dropdown. I will push some changes to revert this issue.
```suggestion
References
----------
```